app/CPReflectionService/RBObject+CocoaPods.rb

```diff
@@ -76,6 +76,12 @@ def self.all_pods
       sources_manager.aggregate.all_pods
     end
 
+    def self.pod_versions(podName)
+      sources_manager.aggregate.search_by_name(podName).first.versions.map do |version|
+        version.to_s
+      end
```

Comment:
`sources_manager.aggregate.search_by_name(podName).first.versions.map(&:to_s)` is a tad more elegant 👍